halyard-config/src/main/java/com/netflix/spinnaker/halyard/services/v1/ConfigService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.services.v1;
+
+import com.netflix.spinnaker.halyard.config.v1.HalconfigParser;
+import com.netflix.spinnaker.halyard.model.v1.Halconfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConfigService {
+  @Autowired
+  HalconfigParser halconfigParser;
+
+  public Halconfig config() {
```

Comment:
Did you avoid naming this `getConfig` so it would be more apparent that there might be some side-effect from calling it?